### PR TITLE
Extend logger with common logging phrases

### PR DESCRIPTION
**Pull request by slemrmartin**
Adding logging phrases common to all collectors to unify logging output

- `logger.collecting`
- `logger.sweeping`
- `logger.collecting_error`

---

**Used by**:

- https://github.com/RedHatInsights/topological_inventory-ansible_tower/pull/91
- https://github.com/RedHatInsights/topological_inventory-amazon/pull/63
- https://github.com/RedHatInsights/topological_inventory-azure/pull/26
- https://github.com/RedHatInsights/topological_inventory-openshift/pull/122

---

**Gem version: 1.0.2**
- [ ] Released to **RubyGems**

---

**TPINVTRY-938**

### CODE REVIEW

**Review by lindgrenj6 — APPROVED** [review]
I like this, makes for a lot less duplication across the services.

**Review by roliveri — APPROVED** [review]

**Review by syncrou — COMMENTED** [review]
This is looking good. I wouldn't mind seeing specs around this.

**Comment by slemrmartin** [comment]
@syncrou https://github.com/RedHatInsights/topological_inventory-providers-common/pull/23 should be merged first

**Review by syncrou — APPROVED** [review]
LGTM 👍 - Still trying to see why Hakiri is red after #23 was merged. Will merge ( and release ) once that is figured out.

**Comment by slemrmartin** [comment]
@syncrou rebase helped

**Pull request merged**
